### ISSUE TITLE
How do we want to instruct for inputdata? 

### ISSUE THREAD

**Issue by sunnivin** [MetOs-UiO/CTSM-Norway-Documentation#12]
How should we properly instruct our team to reference the input-data folders in the [run](https://metos-uio.github.io/CTSM-Norway-Documentation/run) documentation? 

## Current instructions: 

The first time, or whenever your `$WORKDIR` is cleaned, you need to link your working directory with your input data

    [~/CTSM_ROOT]$ cd cime/scripts
    [~/CTSM_ROOT/cime/scripts]$ ./link_dirtree $CESM_DATA /work/users/$USER/inputdata

I feel this is not following our recommended practices. Where do we distribute the `link_tree`-script?

**Comment by sunnivin** [comment]
Another argument for not keeping the `link_dirtree` in our document is that it is not supported anymore, as pointed out by @devarajun in #14.

**Comment by devarajun** [comment]
The other way to instruct pointing out the input data in user_nl_clm. Maybe we need to come up with a better way instead of manually adding in user_nl_clm. How is it done for site simulations @huitang-earth?

**Comment by huitang-earth** [comment]
@sunnivin, This is definitely a question for further discussion. Since link_tree is not supported anymore, I don't think we should use it. Instead, we should guide people to use the inputdata folders properly. Here are my suggestions:
(1) Global clm simulations: the inputdata will be "/cluster/shared/noresm/inputdata/". If there are some missing input, please contact someone (e.g., Kjetil or Sunniva or others) to put the data in place. 
(2) Regional and single cell clm simulations: usually people have to generate the inputdata by themselves. So, we could suggest that they put all the inputdata in "/work/users/$USER/inputdata" for running the model, but for long-term storage, please copy the folder to either /cluster/projects/$project-number/ or NIRD.

Key thing here is that all the inputdata should have a backup after the simulation is done to make sure that the simulations are reproducible.

**Comment by sunnivin** [comment]
> @sunnivin, This is definitely a question for further discussion. Since link_tree is not supported anymore, I don't think we should use it. Instead, we should guide people to use the inputdata folders properly. Here are my suggestions:
> (1) Global clm simulations: the inputdata will be "/cluster/shared/noresm/inputdata/". If there are some missing input, please contact someone (e.g., Kjetil or Sunniva or others) to put the data in place.
> (2) Regional and single cell clm simulations: usually people have to generate the inputdata by themselves. So, we could suggest that they put all the inputdata in "/work/users/$USER/inputdata" for running the model, but for long-term storage, please copy the folder to either /cluster/projects/$project-number/ or NIRD.
> 
> Key thing here is that all the inputdata should have a backup after the simulation is done to make sure that the simulations are reproducible.

I totally agree. I'm thinking that this is a good topic for our team-meeting tomorrow?

**Comment by sunnivin** [comment]
After discussing this topic at our weekly CTSM-Norway meeting we agreed that we will include the comments here made by @huitang-earth, and wait with finishing this feature until the NorESM-group has clarified how they would like to organize their group ownerships.
